pkg/render/logstorage_test.go

```diff
@@ -495,6 +479,7 @@ var _ = Describe("Elasticsearch rendering tests", func() {
 
 				createResources, _ := component.Objects()
 
+				createResources = append([]client.Object{ls}, createResources...)
```

Comment:
We don't need to include `ls` here, instead we should stop checking that it is created.